src/app/examples/segmented-control-example/segmented-control-example.component.html

```diff
@@ -0,0 +1,11 @@
+<kirby-segmented-control [segmentItems]="segmentItems" (segmentClicked)="onSegmentClick($event)"></kirby-segmented-control>
```

Comment:
Just use `items` (same as list, segmented chip control etc):
```suggestion
<kirby-segmented-control [items]="items" (segmentClicked)="onSegmentClick($event)"></kirby-segmented-control>
```